data/fwupd.conf.md

```diff
@@ -0,0 +1,238 @@
+
+FILE FORMAT
+===========
+
+The file consists of a multiple sections with optional parameters. Parameters are of the form:
+
+    [section]
+    key = value
+
+The file is line-based, each newline-terminated line represents either a comment, a section name or a parameter.
+
+Section and parameter names are case sensitive.
+
+Only the first equals sign in a parameter is significant.
+Whitespace before or after the first equals sign is discarded as is leading and trailing whitespace in a parameter value.
+Internal whitespace within a parameter value is retained.
+
+Any line beginning with a hash (`#`) character is ignored, as are lines containing only whitespace.
+
+The values following the equals sign in parameters are all either a string (no quotes needed) or a boolean, which may be given as **true** or **false**.
+Case is not significant in boolean values, but is preserved in string values. Some items are also numeric.
+
+DAEMON PARAMETERS
+=================
+
+The `[daemon]` section can contain the following parameters.
+
+**DisabledDevices=**[GUID];[GUID]
+
+    Allow blocking specific devices by their GUID, using semicolons as delimiter.
+
+**DisabledPlugins=**[PLUGIN_NAME];[PLUGIN_NAME]
+
+    Allow blocking specific plugins by name.
+    Use **sudo fwupdtool get-plugins** to get the list of plugins.
+
+**ArchiveSizeMax=**[SIZE_IN_MB]
+
+    Maximum archive size that can be loaded in Mb, with **0** for the default.
+
+**IdleTimeout=**[IDLE_TIME_IN_SECONDS]
+
+    Idle time in seconds to shut down the daemon.
+    NOTE: some plugins might inhibit the auto-shutdown, for instance thunderbolt.
+
+    A value of **0** specifies "never".
+
+**VerboseDomains=**[DOMAIN]
+
+    Comma separated list of domains to log in verbose mode.
+    If unset, no domains are set to verbose.
+    If set to "*", all domains are verbose, which is the same as running the daemon with **--verbose --verbose**.
+
+**UpdateMotd=**[BOOL]
+
+    Update the message of the day (MOTD) on device and metadata changes.
+
+**EnumerateAllDevices=**[BOOL]
+
+    For some plugins, enumerate only devices supported by metadata.
+
+**ApprovedFirmware=**[HASH];[HASH];[HASH]
+
+    A list of firmware checksums that has been approved by the site admin
+    If unset, all firmware is approved.
+
+**BlockedFirmware=**[HASH];[HASH];[HASH]
+
+    Allow blocking specific devices by their cabinet checksum, either SHA-1 or SHA-256.
+
+**UriSchemes=**
+
+    Allowed URI schemes in the preference order; failed downloads from the first scheme will be retried with the next in order until no choices remain.
+    If unset or no schemes are listed, the default will be: **file,https,http,ipfs**.
+
+**IgnorePower=**[BOOL]
+
+    Ignore power levels of devices when running updates.
+
+**OnlyTrusted=**[BOOL]
+
+    Only support installing firmware signed with a trusted key.
+    Do not enable this on a production or trusted system.
```

Comment:
You mean disable

Reply:
I've changed this to *Do not set this to `false` on a production or trusted system.* to make it clear.